scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -546,16 +576,17 @@ What similarities do you think the integrated UMAP will have to this plot?
 
 ### Integration with `fastMNN`
 
-Finally, we're ready to integrate!
+Finally, we're ready to integrate.
 To start, we'll use the `fastMNN` approach from the Bioconductor [`batchelor` package](http://www.bioconductor.org/packages/3.16/bioc/html/batchelor.html).
 
-`fastMNN` takes as input the `merged_sce` object to integrate, and the first step it performs is actually to run `batchelor::multiBatchPCA()` on that SCE!
-It then uses that PCA matrix to perform the actual batch correction.
+`fastMNN` takes as input the `merged_sce` object to integrate, and the first step it performs is actually to run `batchelor::multiBatchPCA()` on that SCE.
+It then uses that batch-weighted PCA matrix to perform the actual batch correction.
 The `batch` argument is used to specify the different groupings within the `merged_sce` (i.e. the original sample that each cell belongs to), and the `subset.row` argument can optionally be used to provide a vector of high-variance genes that should be considered for this PCA calculation.
 `fastMNN` will return an SCE object that contains a batch-corrected PCA.
+Let's run it and save the result to a variable called `integrated_sce`.
 
 
-```{r run_fastmnn}
+```{r run_fastmnn, live = TRUE}
 integrated_sce <- batchelor::fastMNN(
   # the merged SCE object
```

Comment:
Just a note that without an initial comment, the `# the merged SCE object` will be the first comment in the enlivened chunk, which may look odd, so I suggest adding one:
````suggestion
```{r run_fastmnn, live = TRUE}
# integrate with fastMNN
integrated_sce <- batchelor::fastMNN(
  # the merged SCE object
````